lib/settings_base.py

```diff
@@ -1230,6 +1230,10 @@ def JINJA_CONFIG():
 CEF_PRODUCT = "amo"
 
 # CSP Settings
+
+PROD_CDN_HOST = 'https://addons.cdn.mozilla.net'
```

Comment:
do you want the `PROD_` prefix if it's the default setting? Also, do we need this in the default settings?